app/views/admin/project_groups/_modal_content.html.slim

```diff
@@ -12,8 +12,8 @@ div.modal-body
 
     = error_notification(f, dont_show_default: true)
 
-    - if @entry.parent.present?
-      = f.input :parent_id, collection: [@entry.parent], label_method: :summary, disabled: true
+    = f.input :parent_id, collection: @parents.map { |i| [indented_option_label(i, :summary_or_none), i.id] },
+      include_blank: false
```

Comment:
One thing, I think `i` doesn't make sense as a loop variable. Perhaps `p` for parent or `g` for group?